src/renderer/common/OptionsPanel.tsx

```diff
@@ -156,11 +162,36 @@ const OptionsPanel = () => {
             Launching
           </Box>
         }
-        onClick={onLaunch}
+        onClick={(event) => setAnchorEl(event.currentTarget)}
       >
         Launch
       </LoadingButton>
 
+      <Popover
+        open={Boolean(anchorEl)}
+        anchorEl={anchorEl}
+        onClose={handleClose}
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'center'
+        }}
+        transformOrigin={{
+          vertical: 'top',
+          horizontal: 'center'
+        }}
+      >
+        <Box sx={{ display: 'flex', flexDirection: 'column' }}>
+          <Typography sx={{ p: 2, color: 'black', backgroundColor: 'white' }}>
+            Please make sure to accept the certificates of the browser.
```

Comment:
Refer this link here as well:

https://etherealengine.github.io/etherealengine-docs/docs/devops_deployment/microk8s_linux#accept-invalid-certs

> Put this link in `Endpoints.ts` file and then user it here.

i.e. 
```
Please make sure to accept the certificates of the browser.

Reference: accept-invalid-certs <- this should be a hyperlink
```